client/src/pages/DonationFormContactInfoPage.tsx

```diff
@@ -0,0 +1,212 @@
+            nextButtonText="Next"
+            onNextPage={onNext}
+            pageName="Contact Information"
+            pageNumber={1}
+            pageInstructions="Please fill out your contact information so that The Pregnancy Centre can notify you about donation approval and drop off details."
+            steps={props.steps}
+        >
+            <div className="contact-info-form">
+                <FormItem
+                    className="first-name-field"
+                    formItemName="First Name *"
+                    errorString={firstNameError}
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    inputComponent={
+                        <TextField
+                            input={firstName}
+                            isDisabled={false}
+                            isErroneous={firstNameError.length !== 0}
+                            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                onFirstNameChange(e.target.value);
+                            }}
+                            name="firstName"
+                            placeholder="e.x. Jane"
+                            type="text"
+                            showRedErrorText={true}
+                        />
+                    }
+                />
+                <FormItem
+                    className="last-name-field"
+                    formItemName="Last Name *"
+                    errorString={lastNameError}
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    inputComponent={
+                        <TextField
+                            input={lastName}
+                            isDisabled={false}
+                            isErroneous={lastNameError.length !== 0}
+                            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                onLastNameChange(e.target.value);
+                            }}
+                            name="lastName"
+                            placeholder="e.x. Doe"
+                            type="text"
+                            showRedErrorText={true}
+                        />
+                    }
+                />
+                <FormItem
+                    className="phone-number-field"
+                    formItemName="Phone Number *"
+                    errorString={phoneNumberError}
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    inputComponent={
+                        <TextField
+                            input={phoneNumber}
+                            isDisabled={false}
+                            isErroneous={phoneNumberError.length !== 0}
+                            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                onPhoneNumberChange(e.target.value);
```

Comment:
the design specfies to format the phone number as the user types, do we need to implement this or is it too much?

Reply:
LOL i talked to amanda and we're scrapping that requirement

Reply:
Sorry, I'll update the design specs rn